hack/run-e2e-local.sh

```
@@ -1,290 +1,178 @@
-# Map the registry in the nodes to the docker container
-for node in $(kind get nodes --name "${CLUSTER_NAME}"); do
-  kubectl annotate node "${node}" "kind.x-k8s.io/registry=localhost:${REGISTRY_PORT}" --overwrite
+for bin in crc jq make "$CONTAINER_TOOL"; do
+  command -v "$bin" >/dev/null 2>&1 || fail "Missing required dependency: $bin"
 done
 
-echo "Waiting for node ready..."
-kubectl wait --for=condition=Ready nodes --all --timeout=60s
+if [[ -f "$HOME/.crc_env" ]]; then
+  # shellcheck disable=SC1090
+  source "$HOME/.crc_env"
```

Shellcheck (0.11.0)

[info] Not following: ./.crc_env was not specified as input (see shellcheck -x).

(SC1091)